{
  Throwable exceptionFromLazyResolveLogValidation=null;
  try {
    File expectedFile=getLazyLogFile(testDataFile);
    KotlinTestUtils.assertEqualsToFile(expectedFile,lazyOperationsLog.getText(),HASH_SANITIZER);
  }
 catch (  Throwable e) {
    exceptionFromLazyResolveLogValidation=e;
  }
  return exceptionFromLazyResolveLogValidation;
}
